Cannot build Date

I get the following on CentOS. 
```
[ 51%] Creating directories for 'Date'
[ 52%] Performing download step (git clone) for 'Date'
Cloning into 'Date'...
Already on 'master'
[ 53%] Performing update step for 'Date'
warning: CRLF will be replaced by LF in test/tz_test/tzdata2015e.txt.zip.
The file will have its original line endings in your working directory.
warning: CRLF will be replaced by LF in test/tz_test/tzdata2015f.txt.zip.
The file will have its original line endings in your working directory.
warning: CRLF will be replaced by LF in test/tz_test/tzdata2016c.txt.zip.
The file will have its original line endings in your working directory.
warning: CRLF will be replaced by LF in test/tz_test/tzdata2016d.txt.zip.
The file will have its original line endings in your working directory.
warning: CRLF will be replaced by LF in test/tz_test/tzdata2016e.txt.zip.
The file will have its original line endings in your working directory.
warning: CRLF will be replaced by LF in test/tz_test/tzdata2016f.txt.zip.
The file will have its original line endings in your working directory.
warning: CRLF will be replaced by LF in test/tz_test/tzdata2015e.txt.zip.
The file will have its original line endings in your working directory.
warning: CRLF will be replaced by LF in test/tz_test/tzdata2015f.txt.zip.
The file will have its original line endings in your working directory.
warning: CRLF will be replaced by LF in test/tz_test/tzdata2016c.txt.zip.
The file will have its original line endings in your working directory.
warning: CRLF will be replaced by LF in test/tz_test/tzdata2016d.txt.zip.
The file will have its original line endings in your working directory.
warning: CRLF will be replaced by LF in test/tz_test/tzdata2016e.txt.zip.
The file will have its original line endings in your working directory.
warning: CRLF will be replaced by LF in test/tz_test/tzdata2016f.txt.zip.
The file will have its original line endings in your working directory.
No rebase in progress?
error: Your local changes to the following files would be overwritten by merge:
	test/tz_test/tzdata2015e.txt.zip
	test/tz_test/tzdata2015f.txt.zip
	test/tz_test/tzdata2016c.txt.zip
	test/tz_test/tzdata2016d.txt.zip
	test/tz_test/tzdata2016e.txt.zip
	test/tz_test/tzdata2016f.txt.zip
Please, commit your changes or stash them before you can merge.
Aborting
Index was not unstashed.
CMake Error at /SAN/inm/tools/build_cluster/SIRF-SuperBuild/Release-cuda12.0/builds/Date/tmp/Date-gitupdate.cmake:203 (message):
  
  Failed to rebase in:
  '/SAN/inm/tools/build_cluster/SIRF-SuperBuild/Release-cuda12.0/sources/Date'.
  Output from the attempted rebase follows:
  Cannot rebase: You have unstaged changes.
  Please commit or stash them.
  You will have to resolve the conflicts manually

make[2]: *** [CMakeFiles/Date.dir/build.make:119: builds/Date/stamp/Date-update] Error 1
```

Probably missing `.gitattributes`. I have no idea if this is a new or old bug, as we are on `master` apparently.
https://github.com/SyneRBI/SIRF-SuperBuild/blob/bec271e5f23252a6aee1231a0d165d7e03b8f1d3/version_config.cmake#L186C22-L187

--- a/version_config.cmake
+++ b/version_config.cmake
@@ -183,8 +183,10 @@ set(DEFAULT_RocksDB_TAG v6.26.0)
 set(DEFAULT_mrd-storage-server_URL https://github.com/ismrmrd/mrd-storage-server)
 set(DEFAULT_mrd-storage-server_TAG origin/main)
 
-set(DEFAULT_Date_URL https://github.com/HowardHinnant/date)
-set(DEFAULT_Date_TAG master)
+#set(DEFAULT_Date_URL https://github.com/HowardHinnant/date )
+#set(DEFAULT_Date_TAG master)
+set(DEFAULT_Date_URL https://github.com/SyneRBI/date )
+set(DEFAULT_Date_TAG fixgitattributes)
 
 set(DEFAULT_pugixml_URL https://github.com/zeux/pugixml)
 set(DEFAULT_pugixml_TAG v1.13)

